### PR TITLE
fix for the Service name

### PR DESCRIPTION
**Pull request by udaydenduluri33**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/EUI-8534

### Change description ###

Service names for Global search in a readable format

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x ] No
```

### CODE REVIEW

File: api/globalSearch/index.ts [hunk 1]
```diff
@@ -58,14 +59,17 @@ export function generateServices(jurisdictions: Jurisdiction[]): GlobalSearchSer
...
+
   // Generate global search services
   const globalSearchServices: GlobalSearchService[] = [];
   globalSearchServiceIdsArray.forEach((serviceId) => {
     const jurisdiction = jurisdictions ? jurisdictions.find((x) => x.id === serviceId) : null;
     if (jurisdiction) {
       globalSearchServices.push({ serviceId: jurisdiction.id, serviceName: jurisdiction.name });
     } else {
-      globalSearchServices.push({ serviceId, serviceName: serviceId });
+      const serviceName = serviceNamesMappings.find((map) => map.serviceId === serviceId).serviceName;
```
Comment (Daniel-Lam) [inline api/globalSearch/index.ts:71]: @udaydenduluri33 - what happens if `find()` returns `null`? Would it not cause an error?

**Review by Daniel-Lam — CHANGES_REQUESTED** [review]
@udaydenduluri33 - just one minor observation raised

File: api/globalSearch/index.ts (hunk 1 above)
Comment (Daniel-Lam) [reply api/globalSearch/index.ts:71]: Thanks, @udaydenduluri33

Comment (udaydenduluri33) [reply api/globalSearch/index.ts]: Fixed it now. Thanks

**Review by Daniel-Lam — APPROVED** [review]

**Review by DavidJayakumar — DISMISSED** [review]
LGTM

**Review by connorpgpmcelroy — DISMISSED** [review]
LGTM

**Review by connorpgpmcelroy — DISMISSED** [review]
LGTM

**Review by johnbenjamin-hmcts — APPROVED** [review]

**Pull request closed without merging**